spec/player_spec.rb

```diff
@@ -0,0 +1,12 @@
+require 'player'
+
+describe Player do
+
+ # it "returns two cards for initial hand" do
+ #   output = StringIO.new("[1D, 1D]")
+ #   player_one = PlayerOne.new
+ #   player_one.get_initial_hand
+ #   expect(output.string).to eq("[1D, 1D]")
+ # end
+
```

Comment:
I think once you are more comfortable with dependency inversion and test doubles, writing this test will get easier